pkg/codegen/python/doc.go

```diff
@@ -160,9 +160,11 @@ func (d DocLanguageHelper) GetResourceFunctionResultName(modName string, f *sche
 
 // GenPropertyCaseMap generates the case maps for a property.
 func (d DocLanguageHelper) GenPropertyCaseMap(pkg *schema.Package, modName, tool string, prop *schema.Property, snakeCaseToCamelCase, camelCaseToSnakeCase map[string]string, seenTypes codegen.Set) {
-	if err := pkg.ImportLanguages(map[string]schema.Language{"python": Importer}); err != nil {
-		fmt.Printf("error building case map for %q in module %q", prop.Name, modName)
-		return
+	if _, imported := pkg.Language["python"]; !imported {
+		if err := pkg.ImportLanguages(map[string]schema.Language{"python": Importer}); err != nil {
```

Comment:
I believe in practice this will never hit because we import those [in advance](https://github.com/pulumi/pulumi/blob/dec6a6a49a24e5f015a2f9b7a06bfa43a6a12ac4/pkg/codegen/docs/gen.go#L1774).

Curious if that's true and if you keep this block because `GenPropertyCaseMap` is exported and we avoid some theoretic breaking change.

Reply:
Yeah I was being conservative because of it being exported.

Reply:
@pgavlin Do you think we should remove it?